MekHQ/src/mekhq/campaign/autoResolve/damageHandler/CrewMustSurvive.java

```diff
@@ -0,0 +1,5 @@
+package mekhq.campaign.autoResolve.damageHandler;
```

Comment:
Does this need to be a class, or can we attach it as an enum to one of your other classes?